docs/src/tutorial/readings.rst

```diff
@@ -1,7 +1,7 @@
 Further reading
 ===============
 
-The main documentation is located at http://docs.cython.org/. Some
+The main documentation is located at http://docs.cython.org/en/latest/. Some
```

Comment:
Same here, it's in fact also just backed by readthedocs (but doesn't support https due to the different domain).